app/javascript/Redux/greetingSlice.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
```

Comment:
- [ ] Kindly ensure that your app works. When I try to open the app in the browser I get this error. I have tried compiling the assets but I keeping the error outlining that "axios" does not exist in your `node_modules`, which is true. You did not install the "axios" package (see package.json file) yet you are importing it here.

  In general, please ensure your app actually works. Consider re-cloning the app to possibly see these errors.

  When I try to open app in browser | When I try compile JS
  :---:|:---:
  ![xkxo](https://user-images.githubusercontent.com/36542986/203317362-2d0f4128-8f94-43af-a1d1-6c83e29f8da3.jpg) | ![image](https://user-images.githubusercontent.com/36542986/203317634-0c930208-c71d-43b2-bdb0-4b69dbd6e334.png)